lib/assessments/permission.rb

```diff
@@ -73,23 +77,23 @@ def revoke_level(user)
           assessment.viewers.destroy(user)
         when :network_partner
           assessment.network_partners.destroy(user)
-        assessment.reload
+          assessment.reload
```

Comment:
This statement's giving me pause - I'm going to be sure that this didn't cause any regressions I didn't intend to!